ErpSystemOpgave/ErpSystemOpgave/Data/Customer.cs

```diff
@@ -3,16 +3,30 @@
 
 namespace ErpSystemOpgave.Data;
 
+//eller som record:
+// public record Customer : Person
+// {
+//     public int CustomerId { get; set; }
+
+//     public Customer(string firstName, string lastName, Address address, ContactInfo contactInfo, int customerId) : base(firstName, lastName, address, contactInfo)
+//     {
+//         CustomerId = customerId;
+//     }
+// }
+
 public class Customer : Person
 {
     public int CustomerId { get; set; }
+    public string FullName { get; set; }
+    public string FullAddress { get; set; }
     public DateTime? LastPurchase { get; set; }
 
 
     public Customer(string firstName, string lastName, Address address, ContactInfo contactInfo, int id)
         : base(firstName, lastName, address, contactInfo)
     {
         CustomerId = id;
+        FullName = $"{FirstName} {LastName}";
+        FullAddress = $"{Address.Street}, {Address.ZipCode} {Address.City}";
     }
-
 }
```

Comment:
Let's yeet those comments of mine! 
It is widely agreed upon that bad comment is fuck.

Honestly don't remember what we agreed on with the `FullName` and `FullAddress` properties, but they are also available via the `ToString()` overload in their respective classes.